### PR TITLE
Refactor event service to use constants and env

### PR DESCRIPTION
**Pull request by bzwei**
Allow to set Kafka topic through `ENV['KAFKA_TOPIC']`

Some refactoring to use constants.

### CODE REVIEW

**Comment by hsong-rh** [comment]
LGTM 👍

**Comment by Fryguy** [comment]
bounced the PR to allow Travis and CodeClimate to run now that it's ben merged.

File: app/services/event_service.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 require 'manageiq-messaging'
 
 class EventService
-  APPROVAL_EVENT_TOPIC = 'insights-approval-service'.freeze
+  EVENT_REQUEST_STARTED  = 'request_started'.freeze
```
Comment (mkanoor) [inline app/services/event_service.rb]: @bzwei 
Any reason for using snake case for event names makes it look like a variable. 
Could we just use camel case
**RequestStarted** instead or 
use all upper case with under scores.

**REQUEST_STARTED** which makes it feel like constants from the windows messaging world.

Also can this be stored some place common for other micro services to use.

Comment (bzwei) [reply app/services/event_service.rb]: Did a search on manageiq codebase. All constants are snake case all capitals. 

It is good to store constants in a common place, but we don't have such a place right now.

**Comment by mkanoor** [comment]
👍

**Pull request merged**